File: C++_Base_PT4/XString48.cpp
#include "pt4.h"
using namespace std;

void Solve()
{
    Task("XString48");
    
	string s;
    
    //getline(cin,s);
    
    pt >> s;
    
    char c = s[0];
    
	for(int i = 1; i < s.size(); i++)
	{
		if(s[i] == c)
		{
			s[i] = '.';
		}
		
		if(s[i] == ' ' && s[i+1] != ' ')
		{
			c = s[i+1];
			i++;
		} 
	}
	
	pt << s;
}

File: C#_Base_PT4/XString48.cs
// File: "XString48"
using PT4;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace PT4Tasks
{
    public class MyTask : PT
    {
        public static void Solve()
        {
            Task("XString48");
            string s = GetString();
            char []str = s.ToCharArray();
            char c = str[0];

            int l = s.Length;

            for(int i = 1; i < l; i++)
	        {
		        if(str[i] == c)
		        {
			        str[i] = '.';
		        }
		
		        if(str[i] == ' ' && str[i+1] != ' ')
		        {
			        c = str[i+1];
			        i++;
		        } 
	        }
            s = new string (str);
            Show(str);
            Put(s);
        }
    }
}
